src/instructlab/model/backends/backends.py

```diff
@@ -59,18 +67,26 @@ def __init__(
         model_family: str,
         model_path: pathlib.Path,
         chat_template: str,
-        api_base: str,
         host: str,
         port: int,
+        config: ServerConfig,
     ) -> None:
         self.model_family = model_family
         self.model_path = model_path
         self.chat_template = chat_template if chat_template else CHAT_TEMPLATE_AUTO
-        self.api_base = api_base
         self.host = host
         self.port = port
+        self.config = config
         self.resources: list[Closeable] = []
 
+        # Write logs to a file if log_file is provided
+        if self.config.log_file:
+            self.setup_logs_to_file(logger)
+
+    def setup_logs_to_file(self, a_logger: logging.Logger) -> None:
+        """Write logs to a file if log_file is provided"""
+        write_log_to_file(a_logger, self.config.log_file)
```

Comment:
I am worried about how often this has to be called and how easy it will be to miss a case going forward.  Can you help me understand why this can't be done more globally?

Reply:
Hum, now that I think about it, I could try to add the handler to the root logger instead all the sub-sequent loggers. Agrees that it does not look right 😓

Reply:
Fixed using the root logger :)